src/main/java/frc/robot/commands/ResetRamp.java

```diff
@@ -22,23 +29,28 @@ public ResetRamp(Ramp ramp) {
   @Override
   public void initialize() {
     startTime = Timer.getFPGATimestamp();
+    ramp.setMotor(0.3); //assuming positive is forward with a random speed
```

Comment:
make this a constants

Reply:
Although I don't see a use for this when we don't have it refined yet, I added it